File: hammer-util/src/main/java/dev/vini2003/hammer/util/HUClient.java
package dev.vini2003.hammer.util;

import dev.vini2003.hammer.util.registry.client.HUNetworking;
import net.fabricmc.api.ClientModInitializer;
import org.jetbrains.annotations.ApiStatus;

@ApiStatus.Internal
public class HUClient implements ClientModInitializer {
	@Override
	public void onInitializeClient() {
		HUNetworking.init();
	}
}

File: hammer-util/src/main/kotlin/dev/vini2003/hammer/util/HUClient.kt
package dev.vini2003.hammer.util

import dev.vini2003.hammer.util.registry.client.HUNetworking
import net.fabricmc.api.ClientModInitializer

object HUClient : ClientModInitializer {
	override fun onInitializeClient() {
		HUNetworking.init()
	}
}